pinot-core/src/main/java/org/apache/pinot/core/query/reduce/BrokerReduceService.java

```diff
@@ -70,12 +74,12 @@ public BrokerResponseNative reduceOnDataTable(BrokerRequest brokerRequest, Broke
         queryOptions != null && Boolean.parseBoolean(queryOptions.get(CommonConstants.Broker.Request.TRACE));
 
     ExecutionStatsAggregator aggregator = new ExecutionStatsAggregator(enableTrace);
-    BrokerResponseNative brokerResponseNative = new BrokerResponseNative();
+    BrokerResponseNative response = new BrokerResponseNative();
 
     // Cache a data schema from data tables (try to cache one with data rows associated with it).
-    DataSchema dataSchemaFromEmptyDataTable = null;
-    DataSchema dataSchemaFromNonEmptyDataTable = null;
-    List<ServerRoutingInstance> serversWithConflictingDataSchema = new ArrayList<>();
+    DataSchema schemaOfEmptyTable = null;
+    DataSchema schemaOfNonEmptyTable = null;
+    List<ServerRoutingInstance> serversWithConflictingSchema = new ArrayList<>();
```

Comment:
I think these rewrites are making difficult to follow the PR. Are they actually needed?

Reply:
I'm fine with a rename of the variables to make them easier to read, but given they make this PR so difficult to read, I would encourage you to create another PR with the renames

Reply:
I think this is the only file I refactored this way . Feel free to ignore it.

Reply:
Reverted back to code with super-long var names.

Reply:
Thanks. But please feel free to improve it in another PR!